tests/unit/select_test.coffee

```diff
@@ -344,3 +344,22 @@ test 'optionValuePath with nested valuePath', ->
 
   equal obj2, select.get('selection'),
     'The right selection is retrieved'
+
+test 'shouldEnsureVisible controls whether to ensure visibility', ->
+  expect 2
+
+  select = @subject
+    content: ['foo']
+    ensureVisible: ->
+      ok yes, 'ensureVisible is called if shouldEnsureVisible is true'
+  select.set 'highlighted', 'foo'
+
+  select = @subject
```

Comment:
I think here you can just do `select.set('shouldEnsureVisible', true)` instead of create a new instance of it, right?